### PR TITLE
feat(@desktop/toast): Community Token received notification

### PR DESCRIPTION
**Pull request by Cuteivist**
Closes https://github.com/status-im/status-desktop/issues/13086

status-go change https://github.com/status-im/status-go/pull/4515

### What does the PR do

Handle new notification - community token received

* Handle backend notification
* Using temporary notification delegates. They will be replaced in https://github.com/status-im/status-desktop/issues/12515

### Affected areas

Wallet / notifications

### Screenshot of functionality (including design for comparison)

https://github.com/status-im/status-desktop/assets/11396062/643343e8-0262-43c9-b112-64dde2e59486

### CODE REVIEW

File: ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml [hunk 1]
```diff
@@ -278,6 +280,25 @@ Popup {
...
 
+    Component {
+        id: communityTokenReceivedComponent
+
+        ActivityNotificationCommunityTokenReceived {
+
+            readonly property var community : notification ? root.store.getCommunityDetailsAsJson(notification.communityId) : null
+
+            communityName: community ? community.name : ""
+            communityColor: community ? community.color : "black"
```
Comment (caybro) [inline ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml:291]: Is `black` a good default? Shouldn't it rather be `transparent`?

**Review by caybro — CHANGES_REQUESTED** [review]

File: ui/imports/shared/views/AssetsView.qml [hunk 2]
```diff
@@ -93,7 +93,7 @@ ColumnLayout {
             ExpressionFilter {
                 expression: {
                     d.controller.settingsDirty
-                    return d.tokenIsVisible(model.symbol, model.currentCurrencyBalance) && (customFilter.isCommunity ? !!model.communityId : !model.communityId)
+                    return customFilter.isCommunity ? !!model.communityId : !model.communityId
```
Comment (caybro) [inline ui/imports/shared/views/AssetsView.qml:96]: This change doesn't make sense here, pls remove

**Review by caybro — CHANGES_REQUESTED** [review]

File: ui/imports/shared/views/AssetsView.qml (hunk 2 above)
Comment (caybro) [reply ui/imports/shared/views/AssetsView.qml:96]: It breaks hiding explicitly those selected by the user in `Settings/Manage tokens/Assets/

File: ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml (hunk 1 above)
Comment (caybro) [reply ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml:291]: Alright...

File: ui/imports/shared/views/AssetsView.qml (hunk 2 above)
Comment (caybro) [reply ui/imports/shared/views/AssetsView.qml:96]: In other works, you have to factor in the following code:
```qml
if (!d.controller.filterAcceptsSymbol(symbol)) // explicitely hidden
  return false
```

**Review by caybro — APPROVED** [review]
Nice, thanks!

**Review by alaibe — APPROVED** [review]

**Pull request merged**